docs/guide/machine-learning.asciidoc

```diff
@@ -118,6 +118,57 @@ docker run -it --rm elastic/eland \
 
 Replace the `$ELASTICSEARCH_URL` with the URL for your Elasticsearch cluster. For authentication purposes, include an administrator username and password in the URL in the following format: `https://username:password@host:port`.
 
+[discrete]
+[[ml-nlp-pytorch-air-gapped]]
+==== Install models in an air-gapped environment 
+
+You can install models in a restricted or closed network by pointing the 
+`eland_import_hub_model` script to local files. 
+
+To be able to complete the process below, make sure that 
+https://git-lfs.com/[Git Large File Storage] is installed in your system.
+
+1. Select a model you want to use from Hugging Face. Refer to the 
+{ml-docs}/ml-nlp-model-ref.html[compatible third party model] list for more 
+information on the supported architectures. 
+
+2. Clone the selected model from Hugging Face by using the model URL. For 
+example:
++
+--
+[source,bash]
+----
+git clone https://huggingface.co/dslim/bert-base-NER
+----
+This command results in a local copy of 
+https://huggingface.co/dslim/bert-base-NER.
+--
+
+3. Use the `eland_import_hub_model` script with the `--hub-model-id` set to the 
+directory of the cloned model to install it:
++
+--
+[source,bash]
+----
+eland_import_hub_model \
+      --url 'XXXX' \
+      --hub-model-id /PATH/TO/MODEL \
+      --task-type ner --insecure --es-model-id bert-base-ner
+----
+If you use the Docker image to run `eland_import_hub_model` you must bind mount 
+the model directory, so the container can read the files:
+
+[source,bash]
+----
+docker run --mount type=bind,source=/PATH/TO/MODELS,destination=/models,readonly -it --rm elastic/eland \
+    eland_import_hub_model \
+      --url 'XXXX' \
+      --hub-model-id /models/bert-base-NER \
+      --task-type ner --insecure --es-model-id bert-base-ner
```

Comment:
```suggestion
      --task-type ner \
      --es-username elastic --es-password XXX \
      --es-model-id bert-base-ner
```

Same as above, we shouldn't have --insecure in our examples

Reply:
@davidkyle I copied these commands from [this comment](https://github.com/elastic/eland/issues/502#issuecomment-1335193128). Shouldn't it be updated accordingly?